src/codemagic/utilities/auditing/http_request_auditor.py

```diff
@@ -76,7 +77,7 @@ def _serialize_audit_info(self):
                 'content': self._serialize_response_content(),
                 'elapsed': self._response.elapsed.total_seconds(),
             },
-            'timestamp': datetime.utcnow().isoformat(),
+            'timestamp': datetime.now(timezone.utc).isoformat(),
```

Comment:
Is it something to worry about?

```
>>> from datetime import timezone, datetime
>>> datetime.utcnow().isoformat()
'2023-03-02T11:47:16.262548'
>>> datetime.now(timezone.utc).isoformat()
'2023-03-02T11:47:20.887312+00:00'
```

Reply:
Yeah, I thought about it too, but I thought it wouldn't make much of a difference, but I might be wrong... I will give it a more thorough thought soon.

Alternatively, we could keep `utcnow()` here, or ensure that we get the same output as before by using:
`datetime.now(timezone.utc).replace(tzinfo=None).isoformat()`

but that would kind of be weird and beat the purpose of it... I'll recheck soon.

Reply:
All good.